plugins/modules/events.py

```diff
@@ -139,25 +132,24 @@ def run_module():
     }
 
     # List cluster events
-    if module.params.get('action') == "list":
-        if not module.params.get('cluster_id'):
-            module.fail_json(msg="cluster_id is required for list cluster events action")
-        list_params = {}
-        for k in QUERY_PARAMS_LIST:
-            val = module.params.get(k)
-            if val:
-                if isinstance(val, list):
-                    list_params = list_params | {k: ",".join(val)}
-                else:
-                    list_params = list_params | {k: val}
-
-        response = requests.get(f"{API_URL}/events", params=list_params, headers=headers)
-
-        if not response.ok:
-            result = dict(changed=True, response=response.text)
-            module.fail_json(msg="Error listing cluster events", **result)
-
-        result = dict(changed=False, cluster_events=response.json())
+    if not module.params.get('cluster_id'):
```

Comment:
Per the link at https://petstore.swagger.io/?url=https://raw.githubusercontent.com/openshift/assisted-service/master/swagger.yaml#/events/v2ListEvents , it does not look like the cluster_id is a required query parameter. Do you read the same as well?

Reply:
Yes, the API documentation indicates that `cluster_id` is optional, and I confirmed that a GET call works without it...

```
$ curl -w '%{response_code}\n' -H "Content-Type: application/json" -H "Authorization: Bearer $AI_API_TOKEN" https://api.openshift.com/api/assisted-install/v2/events
[]
200
```

I will update to make it optional.